packages/client3/constants/index.ts

```diff
@@ -28,10 +28,13 @@ export const API_BASE_URL = process.env.NEXT_PUBLIC_GCLOUD_FUNCTIONS_URL
   ? process.env.NEXT_PUBLIC_GCLOUD_FUNCTIONS_URL
   : networkName === "mainnet"
   ? "https://us-central1-free-artists.cloudfunctions.net"
-  : networkName === "murmuration"
-  ? "https://murmuration.goldfinch.finance/_gcloudfunctions"
+  : networkName === "aurora_testnet"
+  ? "https://us-central1-free-artists.cloudfunctions.net"
+  : networkName === "localhost"
+  ? "http://localhost:5001/goldfinch-frontends-dev/us-central1"
   : "http://localhost:5001/goldfinch-frontends-dev/us-central1";
```

Comment:
Very minor thing, I think there is no need to check for networkName as localhost, could just default to localhost url
```suggestion
  : "http://localhost:5001/goldfinch-frontends-dev/us-central1";
```

Reply:
Yeap. Fair point. Thanks!